common/djangoapps/track/backends/adaptive_learning.py

```diff
@@ -2,7 +2,7 @@
 
 from __future__ import absolute_import
 
-from functools import reduce
+from functools import reduce  # pylint: disable=redefined-builtin
```

Comment:
@pomegranited 

> Not sure which is better in this case though, forward compatibility or pylint disable.

Not completely sure either, but today I feel like being forward-compatible... :P